src/NHapi.Base/Model/GenericComposite.cs

```diff
@@ -1,69 +1,76 @@
-using System;
-using System.Collections;
-
 namespace NHapi.Base.Model
 {
-	/// <summary> An unspecified Composite datatype that has an undefined number of components, each 
-	/// of which is a Varies.  
-	/// This is used to store Varies data, when the data type is unknown.  It is also 
-	/// used to store unrecognized message constituents.  
-	/// </summary>
-	/// <author>  Bryan Tripp
-	/// </author>
-	public class GenericComposite : AbstractType, IComposite
-	{
-		/// <summary> Returns an array containing the components of this field.</summary>
-		public virtual IType[] Components
-		{
-			get
-			{
-				IType[] ret = new IType[components.Count];
-				for (int i = 0; i < ret.Length; i++)
-				{
-					ret[i] = (IType) components[i];
-				}
-				return ret;
-			}
-		}
+    using System.Collections;
+
+    /// <summary>
+    /// An unspecified Composite datatype that has an undefined number of components, each
+    /// of which is a Varies.
+    /// This is used to store Varies data, when the data type is unknown.  It is also
+    /// used to store unrecognized message constituents.
+    /// </summary>
+    /// <author>Bryan Tripp.</author>
+    public class GenericComposite : AbstractType, IComposite
+    {
+        private ArrayList components;
+        private IMessage message;

```

Comment:
this field is not required from what I can tell `AbstractType` already has a Message property.

Reply:
done